middlewares/retry.go

```diff
@@ -9,6 +9,13 @@ import (
 	"net/http"
 )
 
+var (
+	_ http.ResponseWriter = &ResponseRecorder{}
```

Comment:
sorry don't understand this bit, I don't think I have seen this in go before... what is it doing?